app/controllers/api/v1/books_controller.rb

```diff
@@ -3,6 +3,8 @@ module V1
     class BooksController < ApplicationController
       def index
         render json: serialized_books
+      rescue StandardError
```

Comment:
This is perfect as is!

If we added more controllers to this application we could move the rescue to the ApplicationController, to keep things DRY.

But adding here whilst there is one controller is perfect imo. I like to DRY on the third repetition. [To avoid this warning from Sandi Metz.](https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction)

Reply:
This is great --- "When they altered their point of view from "I must preserve our investment in this code" to "This code made sense for a while, but perhaps we've learned all we can from it," and gave themselves permission to re-think their abstractions in light of current requirements, everything got easier. Once they inlined the code, the path forward became obvious, and adding new features become faster and easier."